plugin-animation/src/main/java/com/mapbox/maps/plugin/animation/HighLevelAnimatorSet.kt

```diff
@@ -7,6 +7,8 @@ internal data class HighLevelAnimatorSet(
   val animatorSet: AnimatorSet
 ) : Cancelable {
 
+  var started = false
+
   override fun cancel() {
     animatorSet.cancel()
```

Comment:
Should we reset `started` to false here?

Reply:
It's a higher level `cancel` function that may be called by user (or may be not) so we need to set this flag in the place where we set it now to make it all work properly.